clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/description/SerializeApplicationDescription/RestoreSnapshotDescription.groovy

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.google.deploy.description.SerializeApplicationDescription
+
+import com.netflix.spinnaker.clouddriver.google.deploy.description.AbstractGoogleCredentialsDescription
+
+class RestoreSnapshotDescription extends AbstractGoogleCredentialsDescription {
+  String applicationName
+  String accountName
+  Long snapshotTimestamp //timestamp of the snapshot to restore
```

Comment:
this pr doesn't include pulling the snapshot from front50 and restoring it, I am working on that now